packages/core/package.json

```diff
@@ -18,10 +18,10 @@
     "tslib": "^2.4.0"
   },
   "peerDependencies": {
-    "react": ">=16.9.0"
+    "react": ">=18.0.0"
```

Comment:
Why do we **need** this to be `>= 18`?

The spicy hooks are used also on few other projects, so if there's no strong reason to upgrade the minimum peer version, I wouldn't do it. Can't we just ensure compatibility with at least both 17 and 18?

18 seems to have some nasty side-effects, so the adoption might be slow.

Reply:
I guess we don't. I will revert it back for now.